cumulusci/tasks/metadata_etl/layouts.py

```diff
@@ -68,3 +68,70 @@ def _create_new_related_list(self, metadata, api_name, related_list):
             elem.append("customButtons", button)
 
         elem.append("relatedList", text=related_list)
+
+
+class AddFields(MetadataSingleEntityTransformTask):
+    entity = "Layout"
+    task_options = {
+        "layout_section": {
+            "description": "Which layout section the fields should be added to, default is Information",
+            "required": False,
+        },
+        "fields": {
+            "description": "Array of field API names to append to specified layout section",
```

Comment:
```suggestion
            "description": "List of field API names to append to specified layout section",
```

I suggest "list" because it's more non-developer friendly.